packages/cli/src/telemetry/index.ts

```diff
@@ -167,11 +168,10 @@ export class Telemetry {
 		}
 	}
 
-	async trackN8nStop(): Promise<void> {
+	@OnShutdown()
```

Comment:
We have different priorities for shutting down. Is it by default in the "last" group? If not, let's add a priority that makes it shut down last

Reply:
Please see [6e6d4b8](https://github.com/n8n-io/n8n/pull/10305/commits/6e6d4b84726cec56554513fd1c1a7163c492e62d)